sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_incidents.py

```diff
@@ -0,0 +1,110 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_incidents.py
+
+DESCRIPTION:
+    This sample demonstrates how to list incidents and root causes.
+
+USAGE:
+    python sample_incidents.py
+
+    Set the environment variables with your own values before running the sample:
+    1) METRICS_ADVISOR_ENDPOINT - the endpoint of your Azure Metrics Advisor service
+    2) METRICS_ADVISOR_SUBSCRIPTION_KEY - Metrics Advisor service subscription key
+    3) METRICS_ADVISOR_API_KEY - Metrics Advisor service API key
+    4) METRICS_ADVISOR_METRIC_ID - the ID of an metric from an existing data feed
+    5) METRICS_ADVISOR_ANOMALY_DETECTION_CONFIGURATION_ID - the ID of an anomaly detection configuration
+    6) METRICS_ADVISOR_ALERT_CONFIGURATION_ID - the ID of an alert configuration
+    7) METRICS_ADVISOR_INCIDENT_ID - the ID of an incident
+    8) METRICS_ADVISOR_ALERT_ID - the ID of an alert
+"""
+
+import os
+
+
+def sample_list_incidents_for_detection_configuration():
+    # [START list_incidents_for_detection_configuration]
```

Comment:
can we add the sphinx snippets to the client as well?

Reply:
I think the question is do we want to merge them before the UX study? :)